model/dependencies.go

```diff
@@ -6,33 +6,37 @@ import (
 )
 
 type dependencyIncluder struct {
-	Project   *Project
-	requester string
-	included  map[TVPair]bool
+	Project                *Project
+	requester              string
+	included               map[TVPair]bool
+	deactivateGeneratedDep map[TVPair]bool
 }
 
 // IncludeDependencies takes a project and a slice of variant/task pairs names
 // and returns the expanded set of variant/task pairs to include all the dependencies/requirements
 // for the given set of tasks.
 // If any dependency is cross-variant, it will include the variant and task for that dependency.
 // This function can return an error, but it should be treated as an informational warning
```

Comment:
nit: Period at the end of this sentence so we know that `activationInfo` is not a continuation.